src/Controls/src/Core/Compatibility/Handlers/FlyoutPage/iOS/PhoneFlyoutPageRenderer.cs

```diff
@@ -124,6 +124,11 @@ public override void ViewDidAppear(bool animated)
 		{
 			base.ViewDidAppear(animated);
 			Page.SendAppearing();
+			if (!_intialLayoutFinished)
+			{
+				_intialLayoutFinished = true;
+				SetInitialPresented();
```

Comment:
`ViewDidAppear()` is much later in the lifecycle. Not a problem, but this could cause a visible delay where the flyout opens after the user sees the page.

Reply:
@PureWeen , PhoneFlyoutPageRenderer is derived from UIViewController, not UIView. Therefore, overriding MovedToWindow isn’t possible, and invoking the MovedToWindow event handler isn’t applicable in this case.